### DISCUSSION TITLE
Last update forces you to reinstall the plugin everytime you open your IDE

### DISCUSSION THREAD

**Discussion by Stay1444** [community/community#11138] [Copilot Conversations] [answered]
Last Copilot update makes that copilot stops working when you start your IDE. If fixes by reinstalling the plugin, but the next time that you open it its broken again. (Rider & IntelliJ Idea)

**Comment by halfspacer** [comment, marked as answer]
Reinstalling doesn't do it for me. No mention of Copilot in Rider's log either.. Simply no completions whatsoever.
Installed the previous version, 1.1.7.363, for now

**Reply by jansorg** [reply]
@Stay14 Thanks for reporting! That's definitely not what should be happening. Does a re-login at `Tools > GitHub Copilot > Login to GitHub` help?
@halfspacer Can you confirm that `1.1.7.363` is working but `1.1.8.421` is broken for you? Is `1.1.7` working immediately after the downgrade or did you have to do something, e.g. login again? Are you using JetBrains Gateway or the IDE's light edit mode by any chance? Thanks!

**Reply by halfspacer** [reply]
1.1.7.363 is working but 1.1.8.421 is indeed broken for me.
Uninstalling 1.1.8 and installing 1.1.7 solved it instantly. Going into the Copilot menu in Rider would ask me to "Refresh", and then show me completions for the current line, but whilst typing there would be no in-line completions.

**Reply by jansorg** [reply]
@halfspacer Thanks for the update! We're looking into this now and should have an fix released soon.

**Reply by Stay1444** [reply]
Relogin does not help, for me reinstalling it fixes it but it breaks after closing rider again its weird. I will install the old version too were it worked perfectly

**Reply by jansorg** [reply]
@halfspacer @Stay14 Today's update (1.1.11) is hopefully fixing this. If you don't mind, could you update and test if it's working again for you? Thank you! https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/157541

If it's not working, could you provide the log file?

Here's how to enable debug logging for the Copilot plugin.
1. Open `Help > Diagnostic Tools > Debug Log Settings...`
2. Add a new line `#com.github.copilot` to enable debug logging for Copilot
3. Execute the action in your IDE which isn't working as expected
4. Open the log file with `Help > Show Log in ...`
5. Review the log file and post the relevant snippets here

**Reply by Stay1444** [reply]
I just updated and its working perfectly (i tried restarting Rider multiple times and it worked everytime perfectly). Thanks

**Reply by jansorg** [reply]
Thank you for testing and confirming! I'm glad that it's working again :smile:
